Revert/Config.java

```diff
@@ -0,0 +1,108 @@
+import java.nio.charset.Charset;
+//import java.nio.file.Files;
+//import java.nio.file.Path;
+import java.io.File;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.regex.*;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class Config {

```

Comment:
Cała klasa - formatowanie:
1. Jedna linia odstępu między blokami kodu jest wystarczająca. Zawsze :)
2. Jeśli masz jakiś blok ograniczony nawiasami klamrowymi powinien on wyglądać następująco:
```
void doSth() {
    int a = 1;
    try {
        doSthElse();
    } catch(Exception e) {
        e.printStackTrace();
    }
}
```
Bardzo poprawia to czytelność kodu.